odxtools/diagnostictroublecode.py

```diff
@@ -27,14 +29,15 @@ def is_temporary(self) -> bool:
         return self.is_temporary_raw == True
 
     @staticmethod
-    def from_et(et_element, doc_frags: List[OdxDocFragment]) -> "DiagnosticTroubleCode":
+    def from_et(et_element: ElementTree.Element,
+                doc_frags: List[OdxDocFragment]) -> "DiagnosticTroubleCode":
         if et_element.find("DISPLAY-TROUBLE-CODE") is not None:
             display_trouble_code = et_element.findtext("DISPLAY-TROUBLE-CODE")
         else:
             display_trouble_code = None
 
-        if et_element.find("LEVEL") is not None:
-            level = et_element.findtext("LEVEL")
+        if (level_str := et_element.findtext("LEVEL")) is not None:
+            level = int(level_str)
```

Comment:
level_str can be in hex?

Reply:
I don't think so: https://www.data2type.de/xml-xslt-xslfo/xml-schema/datentypen-referenz/xs-byte (sorry, I couldn't quickly find a good English source)